samples/snippets/cpp/VS_Snippets_Misc/NVC_MFC_WordPad/cpp/wordpvw.cpp

```diff
@@ -793,7 +793,7 @@ void CWordPadView::OnPaletteChanged(CWnd* pFocusWnd)
 {
 	ForwardPaletteChanged(m_hWnd, pFocusWnd->GetSafeHwnd());
 	// allow the richedit control to realize its palette
-	// remove this if if richedit fixes their code so that
+	// remove this if richedit fixes their code so that
```

Comment:
This is not a good comment, both in terms of casing, punctuation, and clarity, and choice of wording (their?).

Reply:
This is a terrible comment, but I have no clue what they mean here LOL. I'll leave this one as-is